tensorflow/c/kernels.h

```diff
@@ -190,6 +194,15 @@ TF_CAPI_EXPORT TF_Tensor* TF_AllocateOutput(TF_OpKernelContext* context,
                                             int64_t* dims, int num_dims,
                                             size_t len, TF_Status* status);
 
+// Allocates a temporary Tensor of the specified type and shape. The
+// Tensor must not be used after kernel construction is
+// complete. 
```

Comment:
This description is for `OpKernelConstruction::allocate_temp`. Both `OpKernelContext` and `OpKernelConstruction` have `allocate_temp`.
See https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/core/framework/op_kernel.h;l=999
vs
https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/core/framework/op_kernel.h;l=285

Looks like the signature takes `OpKernelContext` which is good, since this is the more frequently used one. But the comment needs to be updated.